kubernetes/adapter.go

```diff
@@ -227,8 +227,8 @@ func (a *Adapter) newIngress(typ IngressType, metadata kubeItemMetadata, host st
 	loadBalancerType = loadBalancerTypesIngressToAWS[loadBalancerType]
 
 	if loadBalancerType == aws.LoadBalancerTypeNetwork {
-		// ensure ipv4 for network load balancers
-		ipAddressType = aws.IPAddressTypeIPV4
+		// ensure both ipv4 and ipv6 for network load balancers
+		ipAddressType = aws.IPAddressTypeDualstack
```

Comment:
We need to understand why this special case was added and maybe we can remove it (or control with a flag) .